webapp/src/components/molecules/RatingStars/RatingStars.tsx

```diff
@@ -0,0 +1,29 @@
+import React, { useState } from 'react';
```

Comment:
This looks like it works! I meant the "card" as a separate component but don't worry about that

Reply:
🙊ack, totally misinterpreted that! I figured the reviews molec is used on other pages, this way it can just be inserted where ever it needs to go.